nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -31,20 +31,30 @@ def __init__(self):
         self.move_arm_task = MoveArmTask()
         self.send_interface_request_task = SendInterfaceRequestTask()
 
+        self.phases = queue.Queue()
+        self.phase_map = {
+            MissionPlanGoal.GO_TO_PATIENT_PHASE: self.go_to_patient_phase,
+            MissionPlanGoal.GO_HOME_BASE_PHASE: self.go_home_base_phase,
+            MissionPlanGoal.TRIAGE_PATIENT_PHASE: self.triage_patient_phase,
+            MissionPlanGoal.GO_TO_STOCK_PHASE: self.go_to_stock_phase,
+            MissionPlanGoal.GET_ITEMS_PHASE: self.get_items_phase,
+            MissionPlanGoal.RETURN_TO_PATIENT_PHASE: self.return_to_patient_phase,
+            MissionPlanGoal.HANDOFF_ITEMS_PHASE: self.handoff_items_phase,
+            MissionPlanGoal.GO_IDLE_PHASE: self.go_idle_phase,
+        }
+
         self.server = actionlib.SimpleActionServer(
             "mission_planner", MissionPlanAction, self.goal_cb, False
         )
         self.server.start()
 
-        self.phases = queue.Queue()
-
     def go_to_patient_phase(self):
         rospy.loginfo("Nightingale Mission Planner going to patient")
         # Assume door is open
         # TODO: first go to doorside then bedside when door opening added
 
         # update to driving screen
-        task_reponse = self.send_interface_request_task.execute(RobotStatus.DRIVING)
+        task_response = self.send_interface_request_task.execute(RobotStatus.DRIVING)
```

Comment:
why is this a change lol

Reply:
I needed to run a full test, and there was another instance where this was spelled wrong that could prevent a full test, so i did a find and replace.